unit2/unit2.ipynb

```diff
@@ -315,7 +315,7 @@
       "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        "epsilon": epsilon,
</pre><p>We need also to remove the epsilon here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/deep-rl-class/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='94'/>

Reply:
done, thanks!